common/scripts/x86_64/CentOS_7/remote/installRedis.sh

```diff
@@ -16,6 +16,9 @@ install_redis() {
   sudo yum install -y epel-release
   sudo yum update -y
   sudo yum install -y redis
+  # Move the config that we've copied earlier to the default
+  # location of the config in CentOS.
+  sudo mv /etc/redis/redis.conf /etc/redis.conf
```

Comment:
Should we not also do systemctl daemon-reload?
I am not sure whether the changes to conf file will be reloaded or not.

Reply:
Hmm, didn't seem to need it why I did the install, but I'll have to confirm as I was experimenting.

Reply:
Actually I think we don't need to to, we're not changing the service definition, but the config that redis loads. So, we should be good.